presqt/exceptions/exceptions.py

```diff
@@ -0,0 +1,15 @@
+class PresQTException(Exception):
```

Comment:
I'm not sure if there is value in have a package named exceptions and then a module also named exceptions.

This might be a case when just a module is needed - or, even putting the exceptions into the __init__.py of the package until a time arises when it makes sense to break them out into different modules.